content/blog/migrating-vsphere-virtual-volumes-to-kubevirt-with-hpe-csi-driver-for-kubernetes.md

```diff
@@ -0,0 +1,432 @@
+- hosts: kubevirt
+  become: yes
+  tasks:
+  - name: Disable MariaDB
+    service:
+      enabled: no
+      state: stopped
+      name: mariadb
+  - name: Remove default MariaDB datadir
+    file: 
+      path: "{{ var_lib_mysql }}"
+      state: directory
+      recurse: true
+
+# Start/enable MariaDB and deploy TestDB on source
+- hosts: legacy
+  become: yes
+  tasks:
+  - name: Set permissions
+    file:
+      path: "{{ var_lib_mysql }}"
+      owner: mysql
+      group: mysql
+      mode: "0755"
+      seuser: system_u
+      serole: object_r
+      setype: mysqld_db_t
+      selevel: s0
+
+  - name: Enable MariaDB
+    service:
+      name: mariadb
+      state: started
+      enabled: true
+
+  - name: Deploy Test_DB
+    shell: mysql < employees_partitioned.sql
+    args:
+      creates: "{{ var_lib_mysql }}/employees"
+      chdir: "{{ ansible_user_dir }}/test_db"
+
+  - name: Test Test_DB
+    shell: mysql -t < test_employees_sha.sql && mysql -t <<< 'select @@hostname as 'VM';'
+    args:
+      chdir: "{{ ansible_user_dir }}/test_db"
+    register: test_db_results
+
+  - name: Show results
+    pause:
+      prompt: "{{ test_db_results.stdout }}"
+      seconds: 10
+```
+
+Another detail in the playbook that might need clarification is that the Test_DB sources is downloaded to the destination VM only because the database queries are needed for validation of the cloning and migration.
+
+The last few output lines of the playbook reveal that the database was deployed and tested properly.
+
+```shell
+...
++---------+--------+
+| summary | result |
++---------+--------+
+| CRC     | OK     |
+| count   | OK     |
++---------+--------+
++----------------+
+| VM             |
++----------------+
+| mariadb-legacy |
++----------------+:
+ok: [mariadb-legacy]
+
+PLAY RECAP *************************************************************************
+mariadb-kubevirt           : ok=6    changed=1    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+mariadb-legacy             : ok=15   changed=2    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
+```
+
+Once the prerequisites have been installed, we can also inspect the hypervisor platform each VM is running on, to ensure what we’re doing is actually being executed where we expect it.
+
+```shell
+$ ansible -m command all -b -a virt-what
+mariadb-legacy | CHANGED | rc=0 >>
+vmware
+mariadb-kubevirt | CHANGED | rc=0 >>
+redhat
+kvm
+```
+
+# Preparing for the first failure
+
+What is being described in this blog is a perfectly canned example. In reality, the first experiment always fails. Maybe even the second, and third. The point is that it should be non-disruptive and easy to recover from the failures to iteratively improve as the migration is taken across the finish line with as short downtime as possible. What happens before the cutover may take days and weeks of planning and risk free test execution.
```

Comment:
Consider replacing the word "short" with "brief" so it reads "with as brief downtime as possible".

Put a hyphen between "risk" and "free" so it says "risk-free test execution".